images/nginx/rootfs/build.sh

```diff
@@ -20,42 +20,42 @@ set -o pipefail
 
 export NGINX_VERSION=1.20.1
 export NDK_VERSION=0.3.1
-export SETMISC_VERSION=0.32
-export MORE_HEADERS_VERSION=0.33
-export NGINX_DIGEST_AUTH=cd8641886c873cf543255aeda20d23e4cd603d05
-export NGINX_SUBSTITUTIONS=bc58cb11844bc42735bbaef7085ea86ace46d05b
-export NGINX_OPENTRACING_VERSION=0.11.0
-export OPENTRACING_CPP_VERSION=1.6.0
+export SETMISC_VERSION=fddc347134e97d22c34dc5189e76b0ec97bf84d7
+export MORE_HEADERS_VERSION=f85af9649b858e21b400a2150a4c7b8ebd36e921
```

Comment:
ok, as there's no tag cut since 2017

Reply:
EDIT: same: https://github.com/openresty/headers-more-nginx-module/compare/v0.33...master

No big changes since the last release